src/state/opamSysInteract.ml

```diff
@@ -129,7 +129,10 @@ let packages_status packages =
             |> OpamSysPkg.Set.of_list
           in
           packages %% installed
-        | _ -> OpamSysPkg.Set.empty
+        | distribution ->
+          Printf.ksprintf failwith
+            "External dependency handling not supported for BSD distribution '%s'."
+            distribution
```

Comment:
It would better to have the global fail back, by restraining the `bds` pattern
```ocaml
    | "bsd" when spv OpamSysPoll.os_distribution = "freebsd" ->
      let installed =
        let installed =
          run_query_command "pkg" ["query"; "%n"]
          |> List.map OpamSysPkg.of_string
          |> OpamSysPkg.Set.of_list
        in
        packages %% installed
      in
      installed, packages -- installed, OpamSysPkg.Set.empty
    | "bsd" when spv OpamSysPoll.os_distribution = "openbsd" ->
      let installed =
        let installed =
          run_query_command "pkg_info" ["-mqP"]
          |> List.map OpamSysPkg.of_string
          |> OpamSysPkg.Set.of_list
        in
        packages %% installed
      in
      installed, packages -- installed, OpamSysPkg.Set.empty
```